### DISCUSSION TITLE
I find that IntelliSence is  incomplete

### DISCUSSION THREAD

**Discussion by MrSteins** [vuejs/language-tools#1067] [Q&A] [answered]
`v-model`：
<img width="327" alt="截屏2022-03-18 下午12 38 56" src="https://user-images.githubusercontent.com/66707682/158938544-ffe5aa79-b1d6-471c-88b7-a21ab4e3ca19.png">
`modifier`:
<img width="382" alt="截屏2022-03-18 下午12 43 17" src="https://user-images.githubusercontent.com/66707682/158938908-db4a3c07-6e7c-46e8-8772-d803aaffe4ea.png">

Do I have to set additional settings？

**Comment by MrSteins** [comment, marked as answer]
It turned out that I installed the extension of `lit-html`

**Reply by JaavierR** [reply]
Hi, this solve your problem? because recently I notice that I don't have intellisense on the event modifiers, or you activate some other config? (Is the first time that I got this problem) Also i don't have any extra extension that may conflict with volar

**Reply by MrSteins** [reply]
> Hi, this solve your problem? because recently I notice that I don't have intellisense on the event modifiers, or you activate some other config? (Is the first time that I got this problem) Also i don't have any extra extension that may conflict with volar

You can disable all extensions and turn them on one by one to troubleshoot the extra extension

**Reply by ga676005** [reply]
> Hi, this solve your problem? because recently I notice that I don't have intellisense on the event modifiers, or you activate some other config? (Is the first time that I got this problem) Also i don't have any extra extension that may conflict with volar

I install Vetur to fix this.

**Reply by JaavierR** [reply]
Thanks! I tried uninstalling all extensions but that doesn't work, and now i will try with Vetur. BUT I find out that volar works perfectly when the script tag is after the template tag, if is before like the new style SFC the autocomplete fails with the intellisense in some cases for example the modifiers, I don't if this is intended or if is a know bug.

**Comment by johnsoncodehk** [comment]
We don't have implement `v-model` completion, you can open a feature request for it.

But we already have modifier completion, if not works to your project, please open a bug report and provide minimal reproduction.
